x-pack/plugins/uptime/public/components/common/charts/duration_line_series_list.tsx

```diff
@@ -30,6 +31,9 @@ export const DurationLineSeriesList = ({ lines }: Props) => (
         yScaleToDataExtent={false}
         yScaleType="linear"
         fit={Fit.Linear}
+        tickFormat={(d) =>
+          monitorType === 'browser' ? `${microToSec(d)} sec` : `${microToMilli(d)} ms`
```

Comment:
I thought you mentioned it was decided to remove the units, since it says the units in the axis label.

Reply:
on tooltip we still want to display unit